### ISSUE TITLE
Kontent Detector Producing False Positives

### ISSUE THREAD

**Issue by joeleonjr** [trufflesecurity/trufflehog#1650]
### TruffleHog Version
`trufflehog 3.52.1`

### Expected Behavior
TH should detect valid Kontent Delivery API Keys and Management API Keys. See detailed documentation here: https://kontent.ai/learn/docs/apis/api-keys

### Actual Behavior
TH currently detects Kontent Environment ID values, which are PUBLIC. It does not detect API Keys. The [URI used for verification](https://github.com/trufflesecurity/trufflehog/blob/5cfbde783f23cce4c51a92ea0915846b9c532ecc/pkg/detectors/kontent/kontent.go#L51) in the detector is supposed to be public (this is how their Headless CMS product delivers content).

**Comment by kashifkhan0771** [comment]
You're right. The current detector only detects environment IDs, which aren’t highly sensitive, though they're necessary for making API calls. The real secret is the API keys, whether it's a personal API key or a management API key. Fortunately, both follow the same pattern. So, to properly verify an API call, we need to detect both the API keys and the environment ID.

**Comment by kashifkhan0771** [comment]
You're right. The current detector only detects environment IDs, which aren’t highly sensitive, though they're necessary for making API calls. The real secret is the API keys, whether it's a personal API key or a management API key. Fortunately, both follow the same pattern. So, to properly verify API Key, we need to detect both the API keys and the environment ID.

**Comment by kashifkhan0771** [comment]
You're right. The current detector only detects environment IDs, which aren’t highly sensitive, though they're necessary for making API calls. The real secret is the API keys, whether it's a personal API key or a management API key. Fortunately, both follow the same pattern.
